pandas/tests/io/parser/common.py

```diff
@@ -402,6 +405,18 @@ def test_read_chunksize(self):
         tm.assert_frame_equal(chunks[1], df[2:4])
         tm.assert_frame_equal(chunks[2], df[4:])
 
+        # with invalid chunksize value:
```

Comment:
Well, there is a test for ``nrows=-1`` and one for ``chunksize=0``... which is already "too negative". So yes, there are tests for the positivity check.